cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -110,37 +110,61 @@ Index provider and configuration can be specified using the `OPTIONS` clause.
 CREATE LOOKUP INDEX [index_name] [IF NOT EXISTS]
 FOR (n)
 ON EACH labels(n)
+[OPTIONS "{" option: value[, ...] "}"]
```

Comment:
Strictly speaking , the `option: value[, ...]` isn't quite correct since there only exists one specific option for lookup and text indexes. Not sure if we want to reflect that here and in the glossary though, or leave it as is and just look at it as a general OPTIONS meta syntax?

Reply:
You can send in an empty config map as well :P It feels better to have the general syntax here and then specify what is allowed instead of restricting it here (also makes it easier to add other things to it :P)

Reply:
Ok, good point with the empty map. Let's keeps it as it is then